nylas/client/restful_models.py

```diff
@@ -46,7 +53,12 @@ def __init__(self, cls, api):
     def create(cls, api, **kwargs):
         object_type = kwargs.get("object")
         cls_object_type = getattr(cls, "object_type", cls.__name__.lower())
-        if object_type and object_type != cls_object_type and object_type != "account":
+        if (
+            object_type
+            and object_type != cls_object_type
+            and object_type != "account"
+            and not _is_subclass(cls, object_type)
```

Comment:
Many Neural objects are objects that extend an existing class (such as `Message`), but unlike `Drafts` the server payload still denotes the `object` key as `message` instead of - say 'signature'. To prevent this from breaking existing functionality, we check if the class being created is the subclass of the inbound payload's `object` value.